Probesys/bileto
# tec: Add project based on basic Symfony skeleton

Changes proposed in this pull request:

- Add the project generated with the `composer create-project symfony/skeleton:"6.1.*"` command
- Remove some of the auto-generated strings (`###> symfony/framework-bundle ###`) in `.gitignore` and `.env`
- Add a README
- Add a basic home page with a test

How to test the feature manually:

1. Setup a Nginx webserver (e.g. https://www.nginx.com/resources/wiki/start/topics/recipes/symfony/)
2. Check the homepage is displayed

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
